### PR TITLE
Home Work 2

### PR DESCRIPTION
**Pull request by TanyaB777**

### CODE REVIEW

File: src/main/java/hw_2/Task1.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        b = 5;
+        c = a/b;
+        System.out.printf("%d/%d=%d\n", a, b, c);
+        double d = 9.5/5;
+        System.out.printf("9.5/5=%.2f\n", d);
+        // %
+        a = 16;
+        b = 3;
+        c = a%b;
+        System.out.printf("%d%%%d=%d\n", a, b, c);
```
Comment (AntBelyi) [inline src/main/java/hw_2/Task1.java]: Do we really need "%%" between %d and %d

File: src/main/java/hw_2/Task1.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        c = a + 2 * ++b;
+        System.out.println("priority");
+        System.out.println(c);
+        c = (a + 2) * ++b;
+        System.out.println(c);
+        //double type
+        double e = 2.0 - 1.1;
+        System.out.println("accuracy for a double type");
+        System.out.println(e);
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_2/Task1.java:76]: Could you add spaces between after and before operators. I mean not "a+b" but "a + b". Could you change it in all places please

File: src/main/java/hw_2/Task2.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+public class Task2 {
+    public static void main(String[] args) {
+        int x = 3;
+        int y = 4;
+        //ver 1
+        System.out.printf("%d %s%s",x+y,x,y);
+        //ver 2
+        //System.out.println(x+y+" "+x+""+y);
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_2/Task2.java:11]: Could you add spaces between after and before operators. I mean not "a+b" but "a + b". Could you change it in all places please

File: src/main/java/hw_2/Task1.java (hunk 1 above)
Comment (TanyaB777) [reply src/main/java/hw_2/Task1.java]: Fixed

File: src/main/java/hw_2/Task2.java (hunk 3 above)
Comment (TanyaB777) [reply src/main/java/hw_2/Task2.java:11]: Fixed

File: src/main/java/hw_2/Task1.java (hunk 2 above)
Comment (TanyaB777) [reply src/main/java/hw_2/Task1.java:76]: Fixed

**Pull request merged**